main.go

```diff
@@ -5,7 +5,7 @@ import (
 )
```

Comment:
Remove the comment that does not add any value to the code. Comments should provide context or explain complex logic, not redundant statements.